charts/nri-bundle/Chart.yaml

```diff
@@ -15,7 +15,7 @@ sources:
   - https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie
   - https://github.com/newrelic/newrelic-infra-operator/tree/master/charts/newrelic-infra-operator
 
-version: 4.7.1
```

Comment:
I would not release this for now

Reply:
Restored: [a717a26](https://github.com/newrelic/helm-charts/pull/874/commits/a717a26c6d7a8ab31e52c3cce3642fd9cf69501a)